xml/System.Runtime.Versioning/ObsoletedInOSPlatformAttribute.xml

```diff
@@ -92,8 +92,8 @@ This attribute is primarily used by OS bindings to indicate APIs that shouldn't
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the additional message provided for the attribute, or null.</summary>
+        <value>Provides additional message about the obsoleted API, mostly suggests a better alternative for the OS platform specified.</value>
         <remarks>To be added.</remarks>
```

Comment:
What do you think of this rewording? I also moved the value to the remarks.
```suggestion
        <summary>Gets the optional additional message provided for the attribute.</summary>
        <value>A string that represents the message, or <see langword="null"/> if there is none.</value>
        <remarks>Provides an additional message about the obsoleted API, mostly to suggest a better alternative for the OS platform specified.</remarks>
```

Reply:
Looks good, thanks